lib/board.rb

```diff
@@ -0,0 +1,38 @@
+# Board of the game
+# Board has board, board_update, available_positions
+#           valid_move and count_moves methods
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+  end
+
+  def board_update(player_input, symbol)
+    @board[player_input.to_i - 1] = symbol
+  end
+
+  def available_positions(move)
+    @board.to_set.include?(move)
+  end
+
+  def valid_move?(move)
```

Comment:
This particular method is better done outside the board. The board is currently heavily laden with too much responsibility.